src/api/libcellml/importedentity.h

```diff
@@ -98,6 +98,16 @@ class LIBCELLML_EXPORT ImportedEntity
 protected:
     ImportedEntity(); /**< Constructor */
 
+    /**
+     * @brief Virtual set import source method to be implemented by derived classes.
+     *
+     * Virtual setImportSource method to allow the units and component classes to
+     * implement their own versions.
+     *
+     * @param importSource The import source to set.
+     */
+    virtual void doSetImportSource(const ImportSourcePtr &importSource);
```

Comment:
Just as a reminder, `doSetImportSource()` will have to be removed from the API at some point. It really has nothing to do here.